packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1892,9 +1893,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             const maxOpSize = this.context.deltaManager.maxMessageSize;
 
             // If in manual flush mode we will trigger a flush at the next turn break
-            let batchBegin = false;
             if (this.flushMode === FlushMode.Manual && !this.needsFlush) {
-                batchBegin = true;
+                // eslint-disable-next-line no-param-reassign
+                opMetadata = { ...opMetadata, batch: true };
```

Comment:
Is it safe to merge this way? (we'll lose whatever "batch" might have been on the metadata passed in, right?)

Reply:
It looks like container-level submit is private and that the publicly exposed submit methods do not pass 'opMetadata'.